modifiers/lb_modifiers.py

```diff
@@ -0,0 +1,185 @@
+                "{http://www.clarin.eu/cmd/}metadataCreator",
+            ]:
+                continue
+            elif (
+                parent.tag
+                == "{http://www.clarin.eu/cmd/}distributionRightsHolderPerson"
+            ):
+                modified = True
+                new_element = lxml.etree.fromstring(
+                    """
+                    <distributionRightsHolderOrganization xmlns="http://www.clarin.eu/cmd/">
+                        <role>distributionRightsHolder</role>
+                    </distributionRightsHolderOrganization>
+                    """
+                )
+                organization_info_element = lxml.etree.fromstring(
+                    self.organization_info_str
+                )
+                new_element.append(organization_info_element)
+                grandparent = parent.getparent()
+                grandparent.replace(parent, new_element)
+            elif parent.tag == "{http://www.clarin.eu/cmd/}licensorPerson":
+                modified = True
+                new_element = lxml.etree.fromstring(
+                    """
+                    <licensorOrganization xmlns="http://www.clarin.eu/cmd/">
+                        <role>licensor</role>
+                    </licensorOrganization>
+                    """
+                )
+                organization_info_element = lxml.etree.fromstring(
+                    self.organization_info_str
+                )
+                new_element.append(organization_info_element)
+                grandparent = parent.getparent()
+                grandparent.replace(parent, new_element)
+            else:
+                raise ValueError(
+                    f"Unexpected person element of type {parent.tag} encountered"
+                )
+        lxml.etree.indent(cmdi_record)
+        return modified
+
+
+class FinclarinPersonToOrganizationModifier(PersonToOrganizationModifier):
+    """
+    Modifier that fixes records where there is a person whose surname is FIN-CLARIN.
+    """
+
+    def __init__(self):
+        organization_info_str = """
+            <organizationInfo>
+                <organizationName xml:lang="en">FIN-CLARIN</organizationName>
+                <organizationShortName xml:lang="en">FIN-CLARIN</organizationShortName>
+                <departmentName xml:lang="en">University of Helsinki</departmentName>
+                <communicationInfo>
+                    <email>fin-clarin@helsinki.fi</email>
+                    <url>http://www.helsinki.fi/fin-clarin</url>
+                    <address>PO Box 24 (Unioninkatu 40)</address>
+                    <zipCode>00014</zipCode>
+                    <city>University of Helsinki</city>
+                    <country>Finland</country>
+                </communicationInfo>
+            </organizationInfo>
+            """
+        super().__init__(
+            person_surname="FIN-CLARIN", organization_info_str=organization_info_str
```

Comment:
To be consistent with lines 132/133:
```suggestion
            person_surname="FIN-CLARIN",
            organization_info_str=organization_info_str,
```

Reply:
That's not in our hands: we decided to use Black for Python, and Black does what it pleases. If we want to change individual practices like that, we need to ditch Black altogether and switch to something else.